core/crypto/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2018"
 
 [dependencies]
 arrayref = "0.3"
-blake2 = "0.9.1"
+blake2 = { package = "near-blake2", git = "https://github.com/near/near-blake2.git", rev = "29a15a" }
```

Comment:
why do we still have blake2 here

Reply:
Fixed in: https://github.com/near/nearcore/pull/4380/commits/951e7670b98fa639e18f29c9f25a6aae02dd665c

Yeah, this one I certainly missed. Forgot to revert it. Good eyes.